tst/EnergyPlus/api/TestAutosizing.py

```diff
@@ -0,0 +1,17 @@
+import sys
+
+from pyenergyplus.api import EnergyPlusAPI
+
+api = EnergyPlusAPI()
+
+heating_ua_sizer = api.autosizing.heating_airflow_ua_sizer()
+
+for elevation in [0.0, 25.0, 50.0]:
```

Comment:
Pretty simple loop, but nice to see it work.  For these different elevations, this produces a fan coil terminal airflow of:

```
Autosizing succeeded! Value = 602.1543652929821 m3/s
Autosizing succeeded! Value = 600.3717049925842 m3/s
Autosizing succeeded! Value = 598.5933205025165 m3/s
```

Although I think it's supposed to be kg/s?  I need to verify the units and such, and I just made up the input argument, so I'm not worried about specifics, just nice to see things working along.